platform/jewel/ui/src/main/kotlin/org/jetbrains/jewel/ui/component/ScrollableContainer.kt

```diff
@@ -49,6 +55,11 @@ private const val ID_CONTENT = "VerticallyScrollableContainer_content"
 private const val ID_VERTICAL_SCROLLBAR = "VerticallyScrollableContainer_verticalScrollbar"
 private const val ID_HORIZONTAL_SCROLLBAR = "VerticallyScrollableContainer_horizontalScrollbar"
 
+public enum class ScrollbarPosition {
```

Comment:
I'm not a fan of this being public. I think there are zero valid use cases for our users to be able to do this; we should only expose this as an internal API like we do for text field scrollable containers.

Reply:
Made it internal and only one (internal) method uses it now